examples/crypto/SP800-108_counter_KDF/README.md

```diff
@@ -0,0 +1,124 @@
+        0xb6 0xeb 0x18 0xc3 0x3a 0x3d 0x89 0x67 0xa7 0xd6
+        ------------------------------------- end -------------------------------------
+
+
+        Deriving a key using SP800-108 CMAC counter mode...Done
+        Key:
+        ----------------------------------- len: 16 -----------------------------------
+        0x01 0x02 0x03 0x04 0x05 0x06 0x07 0x08 0x09 0x0a 0x0b 0x0c 0x0d 0x0e 0x0d 0x0f
+        ------------------------------------- end -------------------------------------
+        Label:
+        ----------------------------------- len: 32 -----------------------------------
+        0x50 0x53 0x41 0x5f 0x41 0x4c 0x47 0x5f 0x53 0x50 0x38 0x30 0x30 0x5f 0x31 0x30
+        0x38 0x5f 0x43 0x4f 0x55 0x4e 0x54 0x45 0x52 0x20 0x53 0x61 0x6d 0x70 0x6c 0x65
+        ------------------------------------- end -------------------------------------
+        Context:
+        ----------------------------------- len: 49 -----------------------------------
+        0x53 0x61 0x6d 0x70 0x6c 0x65 0x20 0x6b 0x65 0x79 0x20 0x63 0x72 0x65 0x61 0x74
+        0x69 0x6f 0x6e 0x20 0x76 0x69 0x61 0x20 0x53 0x50 0x20 0x38 0x30 0x30 0x2d 0x31
+        0x30 0x38 0x72 0x31 0x20 0x43 0x6f 0x75 0x6e 0x74 0x65 0x72 0x20 0x6d 0x6f 0x64
+        0x65
+        ------------------------------------- end -------------------------------------
+        Capacity: 0x2a
+        CMAC derived key:
+        ----------------------------------- len: 42 -----------------------------------
+        0x3c 0x50 0xb5 0x5a 0x13 0xb9 0x49 0xad 0x25 0xb4 0xb4 0x0f 0xc3 0x7f 0x55 0x38
+        0x36 0xb5 0x9f 0xa0 0xd0 0x74 0xb7 0x3c 0x83 0x17 0x6d 0x4c 0x10 0x5f 0xc2 0x17
+        0x83 0x8e 0xc4 0xa1 0xb0 0x7b 0x8a 0xbe 0xa8 0xf1
+        ------------------------------------- end -------------------------------------
+
+
+        Deriving a key using SP800-108 CMAC counter mode...Done
+        Key:
+        ----------------------------------- len: 16 -----------------------------------
+        0x01 0x02 0x03 0x04 0x05 0x06 0x07 0x08 0x09 0x0a 0x0b 0x0c 0x0d 0x0e 0x0d 0x0f
+        ------------------------------------- end -------------------------------------
+        Label:
+        ----------------------------------- len: 32 -----------------------------------
+        0x50 0x53 0x41 0x5f 0x41 0x4c 0x47 0x5f 0x53 0x50 0x38 0x30 0x30 0x5f 0x31 0x30
+        0x38 0x5f 0x43 0x4f 0x55 0x4e 0x54 0x45 0x52 0x20 0x53 0x61 0x6d 0x70 0x6c 0x65
+        ------------------------------------- end -------------------------------------
+        Capacity: 0x2a
+        CMAC without context derived key:
+        ----------------------------------- len: 42 -----------------------------------
+        0xe1 0xec 0xfc 0000 0x1e 0x2e 0x9a 0xdb 0xd0 0x16 0xb3 0xb4 0xf3 0x23 0xce 0000
+        0xc1 0x05 0x82 0xec 0x81 0xe1 0xfc 0x19 0x40 0x47 0x4c 0xa6 0x84 0xf9 0xe5 0x07
+        0xb5 0x8a 0xbd 0x03 0xbc 0xe5 0x23 0x82 0x05 0x11
```

Comment:
Info: I wrote a [Python implementation](https://github.com/Mbed-TLS/mbedtls/pull/8698) and got the same data.